connect_ext_ppr/models/task.py

```diff
@@ -33,6 +34,8 @@ class Task(Model):
     aborted_at = db.Column(db.DateTime(), nullable=True)
     aborted_by = db.Column(db.String(20), nullable=True)
 
+    dr_instance = relationship(DeploymentRequest, foreign_keys="Task.deployment_request")
```

Comment:
[COD-C] In `MarketplaceConfiguration` we have the following approach:
```
class MarketplaceConfiguration:
    ...
    ppr_id = db.Column(db.String, db.ForeignKey(PPRVersion.id))
    ...
    ppr = relationship('PPRVersion', foreign_keys='MarketplaceConfiguration.ppr_id')
```

I would use the same approach here. Whichever option we choose, I would suggest using the same format for all relations in models.

Reply:
Discussed offline. Changes applied.